dotnet/src/Experimental/Assistants/IChatThread.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.SemanticKernel.Experimental.Assistants;
+
+/// <summary>
+/// Represents a thread that contains messages.
+/// </summary>
+public interface IChatThread
```

Comment:
Do we expect customers to implement this?

Reply:
This allows for mocking in the callers application pattern.